pkg/action/action.go

```diff
@@ -235,6 +243,12 @@ func (fa *FrizbeeAction) createPR(ctx context.Context) error {
 
 	fmt.Printf("Branch %s pushed successfully\n", branchName)
 
+	if existingPR != nil && existingPR.GetState() == "open" {
+		fmt.Printf("PR %d already exists. Won't create a new one\n", existingPR.GetNumber())
+		return nil
+	}
```

Comment:
Since there is a force push before this, this block should probably be moved around line 213 to bail early and ensure NOOP.

Reply:
But then we wouldn't update an existing PR would we?

Reply:
Yeah thank for the clarification above!